substrate/frame/migrations/src/tests.rs

```diff
@@ -333,3 +352,92 @@ fn migration_timeout_errors() {
 		assert_eq!(upgrades_started_completed_failed(), (0, 0, 1));
 	});
 }
+
+#[cfg(feature = "try-runtime")]
```

Comment:
Having the tests split by feature is slightly problematic, as we probably dont check both cases in the CI...

Reply:
Yea me neither... its a recurring problem that we have not just with try-runtime but also a lot with the benchmarks feature.  
Maybe adding another job to the test-pipeline that only tests this crate without the feature could work.

Reply:
Added this https://github.com/paritytech/polkadot-sdk/pull/4251/commits/028aa60ed26e6687e39527765fca6f081a49cc85

Let's see if it works 😁

Reply:
Worked